datasets/PetFindermy_Adoption_Prediction/train_config.json

```diff
@@ -1,24 +1,25 @@
 {
-  "output_dir": "./logs_petfinder/",
+  "output_dir": "./logs_petfinder/gating_on_cat_and_num_feats_then_sum_full_model",
```

Comment:
hm, why `gating_on_cat_and_num_feats_then_sum_full_model `?

Reply:
The existing config utilized this. The name is just a reference to the method used to combine the different embeddings obtained from the model. I just standardized the format in which the output dir was named across the three configs.